index.html

```diff
@@ -1,377 +1,312 @@
-                            opacity: 0.9;
-                            border-radius: 10px; color: white; margin-bottom: 5%;">DISPOSABLE PRODUCTS <br> MANUFACTURER</h1>
-                            <h3 >One Stop Shop For Disposable Products In All Industries</h3>
-                          </div>
-                        <!-- </div> -->
-                      </div>
+                <button
+                    class="navbar-toggler"
+                    type="button"
+                    data-bs-toggle="collapse"
+                    data-bs-target="#navbar"
+                    aria-controls="navbarNavDarkDropdown"
+                    aria-expanded="false"
+                    aria-label="Toggle navigation"
+                >
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+
+                <div
+                    id="navbar"
+                    class="collapse navbar-collapse justify-content-md-end"
+                >
+                    <div class="navbar-nav">
+                        <a class="nav-link active" href="">Home</a>
+                        <a class="nav-link" href="spas.html">Our Products</a>
+                        <a class="nav-link" href="contact.html">Contact Us</a>
                     </div>
-                  </div>
                 </div>
+            </div>
+        </nav>
 
+        <div id="my-carousel" class="carousel slide" data-bs-ride="carousel">
+            <div class="carousel-indicators">
+                <button
+                    type="button"
+                    data-bs-target="#my-carousel"
+                    data-bs-slide-to="0"
+                    class="active"
+                    aria-current="true"
+                    aria-label="Slide 1"
+                ></button>
+                <button
+                    type="button"
+                    data-bs-target="#my-carousel"
+                    data-bs-slide-to="1"
+                    aria-label="Slide 2"
+                ></button>
+                <button
+                    type="button"
+                    data-bs-target="#my-carousel"
+                    data-bs-slide-to="2"
+                    aria-label="Slide 3"
+                ></button>
+                <button
+                    type="button"
+                    data-bs-target="#my-carousel"
+                    data-bs-slide-to="3"
+                    aria-label="Slide 4"
+                ></button>
+                <button
+                    type="button"
+                    data-bs-target="#my-carousel"
+                    data-bs-slide-to="4"
+                    aria-label="Slide 5"
+                ></button>
+            </div>
+            <div class="carousel-inner">
+                <div class="carousel-item active">
+                    <img
+                        src="https://raw.githubusercontent.com/Meeha66/images/main/image1.jpg"
+                        class="d-block w-100"
+                    />
```

Comment:
I recommend changing these to local links (`src="Carousel/image1.jpg"`) rather than relying on a CDN that may cause slow loading times. I didn't look at the rest of the repository so I just copied what was in the initial code.